packages/form/test/lion-form.test.js

```diff
@@ -37,4 +37,22 @@ describe('<lion-form>', () => {
       firstName: 'Foo',
     });
   });
+
+  it('works with the native submit event (triggered via a button)', async () => {
+    let isSubmitted = false;
+    const mySubmit = () => {
+      isSubmitted = true;
+    };
```

Comment:
maybe use a spy here and `spy.calledOnce` in the end to make test even more readable and prevent double firing of the event on the lion-form (which I expect to happen if not properly handled)

Reply:
I would also double check other tests and if they test this with `calledOnce`, I'm really afraid this MR can cause double submit errors

Reply:
used sinon now